x/stakeibc/keeper/msg_server_liquid_stake.go

```diff
@@ -17,114 +15,91 @@ import (
 func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake) (*types.MsgLiquidStakeResponse, error) {
```

Comment:
Can you add a comment specifying that this function has to function even if run non-atomically?

Reply:
done